content/en/cloud-computing.md

```diff
@@ -7,21 +7,20 @@ tags: ["infrastructure", "fundamental", ""]
 
 ## What it is
 
-Cloud computing is a model that offers compute resources like CPU, network, and disk capabilities on-demand over the internet. 
-Cloud computing gives users the ability to access and use computing power in a remote physical location. 
-Cloud providers like AWS, GCP, Azure, DigitalOcean, and others all offer third parties 
-the ability to rent access to compute resources in multiple geographic locations.
+Cloud computing is a model that offers compute resources like CPU, network, and disk capabilities on-demand over the internet, allowing users to access and use computing power in a remote physical location. 
+Cloud providers offer third parties the ability to rent access to compute resources in multiple geographic locations.
+Users rent these compute resources in a public or private virtual cloud (PVC).
+In a public cloud, users share physical hardware with other users, whereas in a PVC, the user rents an entire server. 
```

Comment:
I think `Cloud providers offer third parties the ability to rent access to compute resources in multiple geographic locations.` is a duplicated sentence with the sentence above. (`, allowing users to access and use computing power in a remote physical location.)

Also, I think we don't have to introduce `private virtual cloud (PVC)` concept here. Looks like too details. (By the way, I guess PVC is typo of VPC. VPC does not directly mean a Private cloud. It can be served by a Public cloud service.)

What about just mentioning public cloud and private cloud simply.

```suggestion
Cloud computing can be classified into public cloud and private cloud depending on whether the cloud computing-based services are open to the outside.
```

Reply:
Thanks, @seokho-son I was also not happy with the introduction of a private virtual cloud but had no good suggestions. So I like the suggestion :)